napari/_qt/layer_controls/qt_image_controls_base.py

```diff
@@ -64,16 +69,19 @@ def __init__(self, layer):
         self.colormapComboBox = comboBox
 
         # Create contrast_limits slider
-        self.contrastLimitsSlider = QHRangeSlider(
-            self.layer.contrast_limits,
-            self.layer.contrast_limits_range,
-            parent=self,
-        )
+        self.contrastLimitsSlider = QDoubleRangeSlider(Qt.Horizontal, self)
+        self.contrastLimitsSlider.setSingleStep(0.01)
+        self.contrastLimitsSlider.setRange(*self.layer.contrast_limits_range)
+        self.contrastLimitsSlider.setValue(self.layer.contrast_limits)
+        self.clim_popup = None
```

Comment:
Can we add a tooltip here to let the user know that there is a pop up?  I think it would be helpful for those that don't know about the pop up.
Something similar to--
```self.contrastLimitsSlider.setToolTip(trans._('Right click for pop up contrast limits slider.'))```

Reply:
added!